elk/training/ccs_reporter.py

```diff
@@ -166,7 +172,10 @@ def forward(self, x: Tensor) -> Tensor:
         assert self.norm is not None, "Must call fit() before forward()"
 
         raw_scores = self.probe(self.norm(x)).squeeze(-1)
-        return raw_scores.mul(self.scale).add(self.bias).squeeze(-1)
+        if self.config.norm == "leace":
+            return raw_scores.mul(self.scale).add(self.bias).squeeze(-1)
+        else:
```

Comment:
In case we add other normalizers that use the platt scaling parameters, it might be better to do
```
elif self.config.norm == "burns":
    return raw_scores
else:
    raise ValueError(f"Unknown normalization {self.config.norm}.")